packages/docusaurus/package.json

```diff
@@ -99,7 +99,6 @@
     "react-dev-utils": "12.0.0-next.47",
     "react-error-overlay": "^6.0.9",
     "react-helmet": "^6.1.0",
-    "react-loadable": "npm:@docusaurus/react-loadable@5.5.2",
```

Comment:
Work around yarn bug for now.  For more context, see https://discord.com/channels/226791405589233664/914902909311926322/915277479722237953 (assuming you are in the Yarn Discord server).

Reply:
Fixed in https://github.com/yarnpkg/berry/pull/3824